index.js

```diff
@@ -5,14 +5,42 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "purple"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+let profile = document.querySelector('#profile')
+console.log(profile)
+let name = profile.querySelector('h2')
+let nickname = profile.querySelector('em')
+let image = profile.querySelector('img')
+console.log(name, nickname, image)
+name.innerText = traveler.name
+nickname.innerText = traveler.nickname
+image.src = traveler.photo
+image.alt = traveler.name
 
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(function (sighting){
+    let list = document.createElement('li')
+    list.dataset.id = sighting.id
+    list.innerHTML = `<p>${sighting.description}</p>
+    <img src="${sighting.photo}" alt="${sighting.species}"/>
+    <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>`
+
+    console.log(list)
+
+    let animals = document.querySelector('#animals')
+    animals.append(list)
+})
```

Comment:
This is well done! One possible way to improve this performance wise would be to move line 35 to the outer scope, so that we only have to traverse the DOM once in order to find this element with the `animals` id while still having access to that variable in this callback function.